pyproject.toml

```diff
@@ -1,3 +1,14 @@
 [build-system]
-requires = ["setuptools", "wheel", "urllib3<1.26", "conan>=1.22.2", "scikit-build", 
-  "cmake!=3.17.1,!=3.17.0", "ninja", "pybind11>2.4", "Cython>0.27.1", "numpy>1.16.3"]
+requires = [
+  "setuptools",
+  "wheel",
+  "conan>=1.31.2",
+  "scikit-build",
+  "cmake!=3.17.1,!=3.17.0",
+  "ninja",
+  'Cython>=0.27.1',
+  "pybind11>2.6",
```

Comment:
Same for `pybind11`. Maybe we should keep `"pybind11>2.4"` here.

Reply:
Yep, good call. This must of slipped in when I rebased the PR from master to stable/0.7. Fixed in: https://github.com/Qiskit/qiskit-aer/pull/1131/commits/5ce378f2b4b3f3bfb6ffe0907f11ca6f55cc377a